drivers/rawexec/driver.go

```diff
@@ -95,6 +95,10 @@ var (
 		SendSignals: true,
 		Exec:        true,
 		FSIsolation: drivers.FSIsolationNone,
+		NetIsolationModes: []drivers.NetIsolationMode{
+			drivers.NetIsolationModeHost,
+			drivers.NetIsolationModeGroup,
```

Comment:
Does this mean rawexec will have netns support? Or will it just allow you to have a rawexec task that runs on the host in a group with other tasks that share a netns?

Reply:
Yup you can run raw_exec tasks in an alloc network.